src/loader/doc_attr.ml

```diff
@@ -96,6 +97,7 @@ let parse_attribute : Parsetree.attribute -> parsed_attribute option =
       match load_payload attr_payload with
       | Some p -> Some (`Doc p)
       | None -> None)
+  | "hidden" | "ocaml.hidden" -> Some (`Hidden attr_loc)
```

Comment:
We discussed with @jonludlam and found that we don't need the attribute but prefer the tag instead. Do you mind opening a PR for that ?